README.md

```diff
@@ -15,6 +15,8 @@ a back-end API, and front-end interfacing in a clone of AirBnB.
 
 The project currently only implements the back-end console.
 
+Data (python objects) generated are stored in a json file and can be accessed with the help of the json module in python
```

Comment:
It will be great if the "file" can be replaced with "format". The json is a just a format in with a file can be serialized and deserialized. Thanks